source/_static/clipboard-driver.js

```diff
@@ -0,0 +1,36 @@
+// Grab all elements with highlight-shell class - these will contain all shell commands
+var highlightShellElements = document.getElementsByClassName("highlight-shell");
+
+// Loop through elements with highlight-shell class and add clipboard button to each block
+for (x = 0; x < highlightShellElements.length; x++)
+{
+  // Create button element
+  var clipboardButton = document.createElement("button");
+
+  // Create span element and add glyphicon glyphicon-copy classes to span element
+  // Append span element as child of clipboard button
+  // Add clipboard-btn class to the clipboard button element
+  // Add alt and title attribute to the clipboard button
+  var spanElement = document.createElement("span");
+  spanElement.setAttribute("class", "glyphicon glyphicon-copy");
+  spanElement.innerHTML = "";
+  clipboardButton.appendChild(spanElement);
+  clipboardButton.setAttribute("class", "clipboard-btn");
+  clipboardButton.setAttribute("alt", "Click to copy the command block below");
```

Comment:
Sorry, my earlier review mentioned "alt-text", but that specific HTML attr is only applicable to `img` tags. Can you drop this line? I think the following line takes care of what we wanted here!